src/components/Episode.tsx

```diff
@@ -0,0 +1,36 @@
+import { IEpisode } from "../utils/IEpisode";
+import { generateEpisodeCode } from "../utils/generateEpisodeCode";
+import { removeTagsFromSummary } from "../utils/removeTagsFromSummary";
+import { isEpisodeValid } from "../utils/isEpisodeValid";
+
+interface EpisodeProps {
+    oneEpisode: IEpisode;
+}
+
+function Episode({ oneEpisode }: EpisodeProps): JSX.Element {
+    if (isEpisodeValid(oneEpisode)) {
```

Comment:
Causes a huge gap of information on shows that have missing fields in their data. For example go to your app and choose the show "Rival Survival". Instead consider what to do for each missing property, such as a placeholder image for the case of a missing `episode.img.medium` property. Later checks in the JSX.Element are redundant because this expression will be false.